metricflow/dataflow/builder/dataflow_plan_builder.py

```diff
@@ -1339,18 +1348,30 @@ def _build_filter_specs_for_input_measure(
         metric: Metric,
         input_measure: MetricInputMeasure,
         descendent_filter_specs: Sequence[WhereFilterSpec],
-    ) -> Tuple[WhereFilterSpec, ...]:
+    ) -> MetricInputMeasureSpecFilterSet:
```

Comment:
I would love if we could rename some stuff in this area to make it easier to read. This is not actually just building the filter set for the input measure, it's building filters for the aggregated metric. And since that is the only place filters get applied, it's really building filters for the whole metric query. (This was true before your change, too.) Can we rename it accordingly? Maybe something like:
- `_build_filter_specs_for_input_measure()` -> `_build_filter_specs_for_queried_metric()`

Reply:
Decided to keep this for now, until maybe we can rip it out of the input_measure potentially?